Plugins/SpatialGDK/SpatialGDKEditorToolbar/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorGenerateSnapshot.cpp

```diff
@@ -19,7 +19,7 @@ using namespace improbable;
 namespace
 {
 
-using NameToEntityIdMap = worker::Map<std::string, worker::EntityId>;
+using StringToEntityIdMap = worker::Map<std::string, worker::EntityId>;
```

Comment:
Wouldn't name more accurately describe this map than `String`? Would `Path` be better?